lib/generators/rolemodel/good_job/templates/config/initializers/good_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
+  config.good_job = {
+    smaller_number_is_higher_priority: true, # new default in V4 and for all of ActiveJob
+    queues: ENV.fetch('GOOD_JOB_QUEUES', 'pdf:1;-pdf'),
```

Comment:
Should the generator assume an app will have a `pdf` queue?

Reply:
IIRC this was intended mostly as an example that was extracted from several projects which do have a pdf queue. @jdmcleod @timirwin and I had some discussion about whether it should be called something else like `large_job`, or `serial_job`.

Reply:
Our guiding principle was to minimize overrides. `GOOD_JOB_QUEUES` will be used by default (by GoodJob) and typically is set in Heroku config. Rather than have instructions on overriding it, we opted to give a slightly better default, in particular communicating some basic things about queues which are not used/understood AND handle our most common exception case, `pdf` generation using background jobs. I'll add a comment.